### ISSUE TITLE
[Bug]: new 'ggpp' release 0.5.7

### ISSUE THREAD

**Issue by aphalo** [insightsengineering/teal.modules.general#748]
### What happened?

Not really a bug, but I am about to submit 'ggpp' 0.5.7 to CRAN. Using 'revdepcheck' locally, the check of 'teal.modules.general' does not complete (has been now running for nearly 2 hours and continues). You may like to check that the new version of 'ggpp' does not break anything. I do not expect it to break anything. It does depend on 'ggplot2' >= 3.5.0 while your package depends on  'ggplot2' >= 3.4.0.

![image](https://github.com/insightsengineering/teal.modules.general/assets/2743159/e0f2d327-70bb-41d5-b177-55d336d81976)

This is already a retry, so other reverse dependencies had already been checked good the day before.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

**Comment by donyunardi** [comment]
Thanks for the heads up @aphalo 
We'll double check again on our end.

**Comment by averissimo** [comment]
Some unique scenario may have happened with that revdepcheck. I've tried replicating it as well as doing the recommended replacement method of `tools::check_packages_in_dir()` and no errors nor stalls happened (as `revdepcheck` is unmaintained).

- `revdepcheck::revdep_check(dependencies = "Suggests")`
  - Took 40mins with `teal.modules.general` taking around 10mins
- `tools::check_packages_in_dir(".", reverse = list(repos = getOption("repos"), which = "Suggests"))`

Tested with ggpp `0.5.7`, `0.5.8` and `0.5.8-2`

Thanks for reporting and create a new issue if the issue reappears

ps. internally we are doing weekly testing with main version of all dependencies and that is passing: see [max strategy](https://github.com/insightsengineering/teal.modules.general/actions/runs/9824373214/job/27123266232)
